lambdas/authorisation/src/main/java/uk/gov/di/data/lep/Authorisation.java

```diff
@@ -2,10 +2,14 @@
 
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
+import software.amazon.lambda.powertools.logging.Logging;
+import software.amazon.lambda.powertools.tracing.Tracing;
 
-public class Authorisation implements RequestHandler {
+public class Authorisation implements RequestHandler<Object, Object> {
```

Comment:
Is this change needed? Can we be any more specific?

Reply:
This is to stop it shouting at me, but we don't need it (it is a minor tidy)